radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/P2PModule.java

```diff
@@ -114,7 +113,9 @@ protected void configure() {
     eventBinder.addBinding().toInstance(RenewIssuedProxyCertificatesTrigger.class);
 
     bind(AddressBook.class).in(Scopes.SINGLETON);
-    bind(PeersView.class).to(PeerManagerPeersView.class).in(Scopes.SINGLETON);
+    bind(PeerManagerPeersView.class).in(Scopes.SINGLETON);
+    bind(ProxiedPeersView.class).in(Scopes.SINGLETON);
```

Comment:
we're missing a singleton bind for `ProxiedPeersViewService`

Reply:
Fixed